Source/settings/mods.ts

```diff
@@ -51,6 +51,48 @@ module FullScreenPokemon {
                 }
             },
             {
+                name: "Joey's Rattata",
+                enabled: false,
+                events: {
+                    onModEnable: function (mod: ModAttachr.IModAttachrMod): void {
+                        let characters: any = this.GroupHolder.groups.Character;
+
+                        for (let i: number = 0; i < characters.length; i += 1) {
+                            if (characters[i].trainer) {
```

Comment:
Instead of having a loop with an if statement, try:

```typescript
this.GroupHolder.groups.Character
    .filter(/*... */)
    .forEach(/* ...*/);
```

A bit cleaner.